FitNesseRoot/DbFit/AcceptanceTests/JavaTests/TeradataTests/FlowMode/SetUp/content.txt

```diff
@@ -1,7 +1,9 @@
 !|dbfit.TeradataTest|
 
+|Connect Using File|!-TestDbConnectionDbFitTeradata.properties-!|
```

Comment:
Oracle is using the style without `!--!`, in other places we're using the other style... I'm not sure which is better. The initial tought is that there are some caveats when we're using `!--!` - e.g. when parsing `<`, `>` signs - I guess because that goes to the old Fit classes embedded into FitLibrary instead of the newer FitNesse ones.